lib/cmds/start.ts

```diff
@@ -153,7 +154,7 @@ function start(options: Options) {
         path.resolve(outputDir, binaries[GeckoDriver.id].executableFilename(osType)));
   }
   if (downloadedBinaries[IEDriver.id] != null) {
-    if (options[Opt.IE32].getBoolean()) {
+    if (options[Opt.IE32].getBoolean() || options[Opt.IE].getBoolean()) {
```

Comment:
This conditional sets the architecture to 32-bit. By default it is the system's arch which is probably x64.